python/perf-kernels/tune_gemm/tune_gemm.py

```diff
@@ -54,7 +54,7 @@ def get_full_tuning_space():
     block_k_range = [16, 32, 64, 128, 256]
     split_k_range = [1, 2, 4, 5, 6, 8, 10, 12, 16, 18, 24]
     num_warps_range = [1, 2, 4, 8]
-    group_m_range = [1, 4, 8, 16, 32]
+    group_m_range = [1, 2, 4, 8, 16]
```

Comment:
any reason why we get rid of 2, and 32? If we can't enable XCD mapping, we may need 32.

Reply:
Let me add XCD mapping and re-think about the range for group_m then.